glui_clipboard.cpp

```diff
@@ -0,0 +1,303 @@
+/****************************************************************************
+
+  GLUI User Interface Toolkit
+  ---------------------------
+
+     glui_clipboard_x11.cpp - GLUI_Clipboard_Object class
+
+	 This file -is- in the public domain. Circa summer 2017.
+
+      --------------------------------------------------
+
+  Copyright (c) 1998 Paul Rademacher
+
+  This software is provided 'as-is', without any express or implied
+  warranty. In no event will the authors be held liable for any damages
+  arising from the use of this software.
+
+  Permission is granted to anyone to use this software for any purpose,
+  including commercial applications, and to alter it and redistribute it
+  freely, subject to the following restrictions:
+
+  1. The origin of this software must not be misrepresented; you must not
+  claim that you wrote the original software. If you use this software
+  in a product, an acknowledgment in the product documentation would be
+  appreciated but is not required.
+  2. Altered source versions must be plainly marked as such, and must not be
+  misrepresented as being the original software.
+  3. This notice may not be removed or altered from any source distribution.
+
+  This program is -not- in the public domain.
+
+*****************************************************************************/
+
+#include <limits.h>
+
+#ifdef _WIN32
+#include <windows.h>
+#else
+#ifdef GLUI_GLX
+#include <GL/glx.h>
+//Xlib/X crazily pollute the global namespace.
+#include <X11\Xlib.h>
```

Comment:
I'd point out that it's for the maintainer to make decisions about stability of master, not usually the contributors.